### DISCUSSION TITLE
AG blocks app store

### DISCUSSION THREAD

**Discussion by riderz49** [AdguardTeam/AdGuardHome#4441] [Q&A]
I have AG on a RaspPi connected to my network for use as DNS server. With AG enabled, I can’t make any purchases from the app store. With AG disabled, it works as normal. I have several of the most popular blocklists and don’t know which entries are causing the problem. I’ve whitelisted apple.com, but that doesn’t fix the problem. And I’ve reached the limits of my ability to troubleshoot this problem. 

Any ideas?? Please help, you’re my only hope.

**Comment by ainar-g** [comment]
Sorry about the late response.  You should check the _Query log_ page and see which queries get blocked.  From the same page you can unblock them and see if that fixes your issue.
